### ISSUE TITLE
Extension is blank when using redux-form, having nested objects in state and no errors showing

### ISSUE THREAD

**Issue by davidhtien** [zalmoxisus/redux-devtools-extension#88]
my state object looks something like this
```
var initialState = {
    form: {
      firstName: '',
      lastName: '', 
      email: '', 
      sex: '', 
      favoriteColor: '', 
      employed: '', 
      notes: ''
    },
    reducer: {
      trainee: trainee,
      tas: tas,
      events: events,
      rolls: rolls,
      slips: slips,
      eventsSlipsRolls: events_slips_rolls,
      date: new Date(),
      selectedEvents: [],
      unexcusedAbsencesShow: true,
      unexcusedTardiesShow: true,
      excusedShow: true,
      submitRollShow: false,
      submitLeaveSlipShow: false,
      submitGroupLeaveSlipShow: false,
      leaveSlipsShow: false,
      otherReasonsShow: false
    }
  };
```

when I open the extension it's just blank and there's no error so I can't figure out why it's not working

**Comment by zalmoxisus** [comment]
Thanks for the report. An example to reproduce this would be welcome. 

You can try [the example of redux-form](http://redux-form.com/5.0.1/#/examples/simple), which works with the extension.

To have more meaningful errors, [run the extension in dev mode](https://github.com/zalmoxisus/redux-devtools-extension#1-get-the-extension).

**Comment by davidhtien** [comment]
It started working today... not really sure what fixed it

**Comment by zalmoxisus** [comment]
@davidhtien, do you use the default log monitor?

**Comment by zalmoxisus** [comment]
There were some monitors that wasn't able deal with an empty liftedState. Fixed in `v1.0.0.21`. Please give it a try and feel free to reopen the issue if the problem still appears.
